UI/Web/src/app/user-settings/user-preferences/user-preferences.component.html

```diff
@@ -251,6 +252,15 @@ <h2 class="accordion-header">
                                 </div>
 
                                 <div class="row g-0">
+                                    <div class="col-md-6 col-sm-12 pe-2 mb-3">
+                                        <label for="settings-book-reading-style" class="form-label">Writing Style</label>&nbsp; <i class="fa fa-info-circle" aria-hidden="true" placement="right" [ngbTooltip]="bookWritingStyleToolTip" role="button" tabindex="0"></i>
```

Comment:
You need the aria-describeby="settings-book-writing-style-help" so the screenreader understands the tooltip message